packages/cashier/src/pages/account-transfer/account-transfer-form/account-transfer-form.tsx

```diff
@@ -404,95 +404,162 @@ const AccountTransferForm = observer(
-                                            is_large
-                                            label={localize('From')}
-                                            list={from_accounts}
-                                            list_height='404'
-                                            name='transfer_from'
-                                            value={selected_from.value}
-                                            onChange={(e: TReactChangeEvent) => {
-                                                onChangeTransferFrom(e);
-                                                handleChange(e);
-                                                setFieldValue('amount', '');
-                                                setTimeout(() => setFieldError('amount', ''));
-                                            }}
-                                            error={selected_from.error}
-                                        />
-                                        <Dropdown
-                                            id='transfer_to'
-                                            className='account-transfer-form__drop-down account-transfer-form__drop-down--to-dropdown'
-                                            classNameDisplay='cashier__drop-down-display'
-                                            classNameDisplaySpan='cashier__drop-down-display-span'
-                                            classNameItems='cashier__drop-down-items'
-                                            classNameLabel='cashier__drop-down-label'
-                                            classNameHint='account-transfer-form__hint'
-                                            test_id='dt_account_transfer_form_to_dropdown'
-                                            is_large
-                                            label={localize('To')}
-                                            list={to_accounts}
-                                            list_height='404'
-                                            name='transfer_to'
-                                            value={selected_to.value}
-                                            onChange={(e: TReactChangeEvent) => {
-                                                onChangeTransferTo(e);
-                                                setFieldValue('amount', '');
-                                                setTimeout(() => setFieldError('amount', ''));
-                                            }}
-                                            hint={transfer_to_hint}
-                                            error={getMt5Error() ?? selected_to.error}
-                                        />
+                                <>
+                                    <div className='account-transfer-form__inline-warning-message'>
+                                        {!Number(remaining_transfers) && (
+                                            <InlineMessage
```

Comment:
just wanted to ask, why do we have inline message component when we already have something similar which is the hint-box component?

Reply:
Its a new component. I didn't know there was hint-box that did the same thing 😅 
cc: @farzin-deriv

Reply:
Yeah seems to be the same thing but a different name than the design component 🤔 

We can mark `HintBox` as deprecated if you guys agree 🙇🏻 
```ts
/** @deprecated Use `InlineMessage` instead. */
const HintBox = ({
```

Reply:
Yeah, we can comment that out i guess. Shall i do it in this pr?

Reply:
yup best to add the deprecated message so others can migrate to InlineMessage 🙏

Reply:
Yeah we can, It's just a JSDoc comment so it won't affect anything.
Hopefully, after this devs will see the `deprecated` warning and start replacing the component.

Reply:
I will add the comment